trollmoves/movers.py

```diff
@@ -71,7 +71,12 @@ def move_it(pathname, destination, attrs=None, hook=None, rel_path=None):
         raise
 
     try:
-        mover(pathname, new_dest, attrs=attrs).copy()
+        m = mover(pathname, new_dest, attrs=attrs, backup_targets=backup_targets)
+        m.copy()
+        last_dest = m.destination
```

Comment:
Ah! Nice catch!

Reply:
... although this only changes the log message, not the message published by the Client :see_no_evil:

Reply:
One possibility:
* in `movers.move_it()` return `m.destination`
* in `server.RequestManager._move_file()` replace `message.data['destination']` with the return value and return the modified message if the transfer was succesful
  * in the case of error do what is done already
* in `server.RequestManager._move_files()` after the `self._move_file()` call check `message.type is 'err'` instead of `error_message is not None`
* in `server.RequestManager.push()` again check the `message.type`, run `clean_url` if necessary

Reply:
Ah, you are right. Crap.